atd-vze/src/queries/dashboard.js

```diff
@@ -1,92 +1,22 @@
 import { gql } from "apollo-boost";
 
-// These queries are derived from the ETL that populates our Socrata data sets
+// This query is derived from the view that populates our Socrata dataset
 // so that the VZV and VZE widget totals match up
-// See https://github.com/cityofaustin/atd-vz-data/blob/master/atd-etl/app/process/socrata_queries.py
 export const GET_CRASHES_YTD = gql`
-  query GetCrashesYTD($yearStart: date, $yearEnd: date) {
-    fatalities: atd_txdot_crashes_aggregate(
-      where: {
-        crash_date: { _lt: $yearEnd, _gte: $yearStart }
-        private_dr_fl: { _eq: "N" }
-        _and: [
-          { crash_date: { _is_null: false } }
-          { crash_time: { _is_null: false } }
-        ]
-        _or: [
-          { in_austin_full_purpose: { _eq: true } }
-          { _and: [{ city_id: { _eq: 22 } }, { position: { _is_null: true } }] }
-        ]
-      }
+  query GetCrashesYTD($yearStart: String, $yearEnd: String) {
+    socrata_export_crashes_view_aggregate(
+      where: { crash_timestamp: { _lt: $yearEnd, _gte: $yearStart } }
```

Comment:
i think i should update this to less than or equal to here? so if a crash was on `2024-01-01T00:00:00` it would get counted for this year?